libraries/AP_VideoTX/AP_VideoTX.h

```diff
@@ -15,6 +15,13 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
+
+#ifndef HAL_VIDEOTX_ENABLED
+#define HAL_VIDEOTX_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
should default enabled so we don't break existing setups

Reply:
The other option would be to make it dependent on SmartAudio and CRSF but default on is simpler and my preference